package.json

```diff
@@ -15,7 +15,6 @@
     "prop-types": "^15.8.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-dropzone": "^14.2.3",
```

Comment:
why removed?

Reply:
@maa-wtag bhai, used react state to handle the same functionality.